scripts/utils/import-metrics-loop.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+
+# set kubeconfig
+export KUBECONFIG=$1
+
+# set timestamp, create dir for export path, set permissions, navigate
+st1=$(date +"%Y-%m-%d")
+mkdir -p $PWD/metrics-$st1
+chmod +x metrics-$st1
+cd metrics-$st1
+
+selector='{__name__!=""}' 
+
+
+# run script for mimirtool via kubectl on target cluster
+bash -c "./../import-metrics-mimirtool.sh $1" &
+
+#TODO: improve wait
+sleep 5
+
+
+for i in 6 4 2
```

Comment:
Can we indent code within loops and conditional statements so that its easier to identify what is part of the loop/conditional?